libs/tup-components/src/auth/LoginComponent/LoginComponent.tsx

```diff
@@ -124,7 +129,12 @@ const LoginComponent: React.FC<LoginProps> = ({ className }) => {
       </Formik>
       <div className={styles.footer}>
         <p>Having trouble logging in?</p>
-        <a className={styles.link} href="/account/help">
+        <a
+          className={styles.link}
+          href="https://dev.tup.tacc.utexas.edu/about/help/"
```

Comment:
Unless there is some URL re-writing I am unaware of, this URL will not work when deployed to production.

Core-Portal uses a [`url`](https://docs.djangoproject.com/en/4.1/ref/templates/builtins/#url) template tag.[^1] Also, just `/about/help/` should work on up-to-date remote servers.[^2] I defer to to @jarosenb for direction.

[^1]: https://github.com/TACC/Core-Portal/blob/main/server/portal/templates/includes/nav_portal.raw.html#L38
[^2]: As in a dev, staging, or prod that has all the pages it should have for complete testing.